tests/test_cff2pages.py

```diff
@@ -313,6 +313,86 @@ def check_folders(cls, tmp_dir):
     return index_file
 
 
+class DiacriticsCffTester(unittest.TestCase):
```

Comment:
Regarding the documentation, could you add a docstring here and to the other methods?